rpc_util.go

```diff
@@ -840,18 +843,18 @@ func recvAndDecompress(p *parser, s recvCompressor, dc Decompressor, maxReceiveM
 			if err == nil {
 				out = mem.BufferSlice{mem.SliceBuffer(uncompressedBuf)}
 			}
-			size = len(uncompressedBuf)
 		} else {
-			out, size, err = decompress(compressor, compressed, maxReceiveMessageSize, p.bufferPool)
+			out, _, err = decompress(compressor, compressed, maxReceiveMessageSize, p.bufferPool)
```

Comment:
Since `size` isn't used anymore, we should remove the return value in `decompress`. As `decompress` is an internal function, we can freely change it's returns type.

Reply:
Done